README.md

```diff
@@ -99,136 +99,121 @@ All other development dependencies will be installed by running the following fr
 npm install
 ```
 
-### Package Types
+### Build Types
 
-#### Infusion All Build
+#### Distribution Builds
 
-Will include all of Infusion, including third-party dependencies. The source files packaged along with the single
-concatenated JavaScript file will include all of the demos and unit tests. This is a good choice if you are trying to
-learn Infusion.
+Will build a set of predefined distribution bundles of Infusion, some of which include third-party dependencies and some
+of which do not, for distribution on [NPM](https://www.npmjs.com/package/infusion). Each distribution file will be
+placed in the `dist` directory and will be accompanied by a [source map](#source-maps).
 
 ```bash
-npm run build:all
+npm run build
 ```
 
-By default all packages are minified. Running the build script with the `:source` suffix will allow you to maintain
-readable spacing and comments.
+Distribution bundles can be [viewed on unpkg](https://unpkg.com/browse/infusion/dist/).
+
+#### Infusion All Build
+
+Will include all of Infusion, including third-party dependencies. The source files packaged along with the single
+concatenated JavaScript file will include all of the demos, examples and unit tests. This is a good choice if you are
+trying to learn Infusion.
 
 ```bash
-npm run build:all:source
+npm run build:pkg
 ```
 
 #### Custom Build
 
 Will only include the modules you request, and all of their dependencies, minus any that are explicitly excluded. Unlike
-the "all" build, none of the demos or tests are included with a custom package.
+the [Infusion All](#infusion-all-build) build, none of the demos, examples or tests are included with a custom package.
 
 ```bash
-npm run build:custom
-```
-
-By default all packages are minified. Running the build script with the `:source` suffix will allow you to maintain
-readable spacing and comments.
-
-```bash
-npm run build:custom:source
+npm run build:pkg:custom
 ```
 
 #### Custom Build Options
 
-Any of the following options can be passed to a custom build by adding two hyphens and then the appropriate option.
-Examples are shown below.
+Any of the following options can be passed to a custom build by setting environment variables. Examples are shown below.
 
-##### --include
+##### FL_INCLUDE
 
 __value__: "module(s)" (String)
 _only available to custom builds_
 
-The `--include` option takes in a comma-separated string of the [Modules](#modules) to be included in a custom package.
-If omitted, all modules will be included (demos and tests will not be included).
+The `FL_INCLUDE` environment variable takes in a comma-separated string of the [Modules](#modules) to be included in a
+custom package. If omitted, all modules will be included (demos and tests will not be included).
 
 ```bash
-npm run build:custom -- --include="inlineEdit, uiOptions"
+FL_INCLUDE="fluid-inline-edit, fluid-ui-options" npm run build:pkg:custom
 ```
 
-##### --exclude
+##### FL_EXCLUDE
```

Comment:
I probably missed some relevant discussion on this - why did we change from a command line option to an env variable?

Reply:
Simply, this is so to support passing the values down between npm scripts. You can use the `--` notation to pass some arguments down but it gets confusing when a scrips runs multiple other scripts or they go down a few levels. In migrating more functionality to buildModules.js we did remove some of this complexity in the scripts so we might actually be able to use the command line arguments again. I'll explore that.

Reply:
In my latest push I have removed all of the environment variables out of the npm scripts related API. However, they are still optional for use with buildModules.js